src/main/java/at/hannibal2/skyhanni/utils/RegistryUtil.kt

```diff
@@ -0,0 +1,56 @@
+package at.hannibal2.skyhanni.utils
+
+object RegistryUtil {
+
+    private const val PREFIX: String = "REG"
+    private const val SUCCESS_CODE: Int = 0
+    private const val FAILURE_CODE: Int = 1
+
+    fun addKey(runtime: Runtime, location: RegistryPaths, key: String) : Boolean {
```

Comment:
There isn't really a point to pass the `Runtime` around. There will only ever be one of those, so we might as well save us the parameter for all those methods.